airtonix/bankdownloader
# feat: able to download transactions from anz

# Description

- Introduces the processor struct and interface
- Introduces the automation struct for manipulating Playwright
- Introduces a working downloader for ANZ.

`~/Documents/bankdownloader/config.yaml`

```
# yaml-language-server: $schema=https://raw.githubusercontent.com/airtonix/bankdownloader/master/schemas/config.json
dateFormat: "2006-01-02T15:04:05Z07:00"
sources:
  - name: anz
    accounts:
      - name: YOUR ACCOUNT NAME
        number: YOUR ACCOUNT NUMBER
    config:
      format: "Agrimaster(CSV)"
      daysToFetch: 7
      credentials:
```

## Tasks



## What kind of change is this?


## Steps to test changes



1. `just setup`
2. `just build`
3. Run the app with `./bin/bank-downloaders init`
4. modify the configuration file with your banks login details, and a series of accounts
5. Run `./bin/bank-downloaders download`
6. confirm that a file is created in `./downloads`
